models/ed/R/met2model.ED2.R

```diff
@@ -54,12 +55,18 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
   dl <- c(0, 32, 61, 92, 122, 153, 183, 214, 245, 275, 306, 336, 367)
   month <- c("JAN", "FEB", "MAR", "APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC")
   mon_num <- c("01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12")
-  day2mo <- function(year, day) {
-    leap      <- lubridate::leap_year(year)
-    mo        <- rep(NA, length(day))
-    mo[leap]  <- findInterval(day[leap], dl)
-    mo[!leap] <- findInterval(day[!leap], dm)
-    return(mo)
+  day2mo <- function(year, day, leap_year_agnostic) {
+    if(leap_year == FALSE){
+      mo        <- rep(NA, length(day))
+      mo <- findInterval(day, dm)
+      return(mo)
+    }else{
+      leap      <- lubridate::leap_year(year)
+      mo        <- rep(NA, length(day))
+      mo[leap]  <- findInterval(day[leap], dl)
+      mo[!leap] <- findInterval(day[!leap], dm)
+      return(mo)
+    }
```

Comment:
`if (x == FALSE)` can be more efficiently written as `if (!x)`.

Also, `mo <- rep(NA, length(day))` is the same in both blocks, so it can be pulled out of the `if-else` altogether.

Finally, the Tidy style guide technically requests spaces between `if/else` and `(/{` (i.e. `if (...) {` and `} else {`.) 

Personally, I would recommend enabling RStudio's "Diagnostics" highlighting for editing code -- it does a good job of keeping you honest about style things like this. To do that, go to "Tools / Global Options / Editing / Diagnostics" and check the things you want it to check for you (I like to have every box in that section checked).